tools/linker-process/index.bzl

```diff
@@ -0,0 +1,92 @@
+load("@build_bazel_rules_nodejs//:providers.bzl", "JSModuleInfo", "JSNamedModuleInfo")
```

Comment:
Could we add a brief overview of what this file is for?

Reply:
Done.